frontend/src/components/balance/styles.css

```diff
@@ -0,0 +1,40 @@
+.card-balance {
+    background-color: #32774A;
```

Comment:
you did a great job to find the ugliest color. Maybe you could spend just 2 more minutes to find a color that's not so poisonous?